.github/workflows/trivy.yml

```diff
@@ -0,0 +1,20 @@
+name: CI
+
+on:
+  push:
+    branches: ["*"]
+  pull_request:
+    branches: [main]
```

Comment:
I suppose we don't have to check it on events push and pull_request, a corn job should be ok